### ISSUE TITLE
Launching a new site from the site checklist leads to a dead end

### ISSUE THREAD

**Issue by rachelmcr** [Automattic/wp-desktop#681]
#### Steps to reproduce
1. Create a new site in the app.
2. Under My Sites > My Home, complete the site checklist.
3. After selecting "Launch site" (the last step in the checklist), follow the on-screen prompts to finish the site launch.

#### What I expected

I expect a success message and the opportunity to go back to My Home or another part of Calypso.

#### What happened instead

My launched site opened in a web browser and the app was stuck on a screen with this message:

> Hooray! Your site will be ready shortly.

There was no way to exit that screen except to close and reopen the app.

#### App Version / OS version

WPDesktop 4.4.1 beta / macOS 10.15

#### Screenshot / Video

![Screenshot 2019-10-22 11 17 23](https://user-images.githubusercontent.com/8658164/67290354-e0500f80-f4d7-11e9-9fc5-987baf36ef51.png)

#### Context / Source
#manual-testing

**Comment by rachelmcr** [comment]
@Automattic/zelda I just want to make sure this is on your radar so you can prioritize it accordingly. This affects a small number of users: the `wpcom_site_launched` event occurs less than 10 times per day (measured by checking that `browserdocumentreferrer` is `http://127.0.0.1:41050/`) compared to thousands of times per day elsewhere.

**Comment by p-jackson** [comment]
Thank you @rachelmcr!

Adding that this is also broken in the stable channel.
WPDesktop 4.4.0 release / macOS 10.14.6

**Comment by rachelmcr** [comment]
I re-tested this with v4.7.0-beta1 (macOS 10.15.2) and found that selecting "Launch my site" from the site setup checklist now works as expected. It launches my site and redirects me in the app to the customer home (My Site > My Home).

I tested the following scenarios:

* Launching a site on the free plan
* Launching a site with a paid plan purchased during signup
* Launching a site starting on the free plan and purchasing a paid plan during site launch

Closing this as resolved; we can reopen if it turns out there's a scenario/flow that still isn't working as expected.
